frontend/src/components/editor/DescriptionEditor.js

```diff
@@ -3,40 +3,173 @@ import PropTypes from 'prop-types';
+  }
+
+  onUpdate = (type, markdown) => {
+    const { onUpdate } = this.props;
+    const { about, application, prerequisites } = this.state;
+
+    if (['application', 'about', 'prerequisites'].includes(type)) {
+      this.setState({
+        [type]: markdown
+      });
+
+      const description = {
+        application,
+        about,
+        prerequisites,
+        [type]: markdown
+      };
+
+      // add trailing line break if is missing
+      const descriptioText = Object.values(description).reduce(
+        (aggregator, value) => aggregator + (value.endsWith('\n') ? value : `${value}\n`),
+        ''
+      );
+
+      onUpdate(descriptioText, 'spec.description');
+    } else {
+      console.warn('Wrong type use correct one, cannot update');
+    }
+  };
+
+  validate = type => {
+    const { onValidate } = this.props;
+    const markdown = this.state[type];
+
+    this.setState({
+      [`${type}Error`]: DescriptionEditor.validateMarkdown(markdown)
+    });
+
+    onValidate('spec.description');
+  };
+
+  render() {
+    const { about, application, prerequisites, aboutError, applicationError, prerequisitesError } = this.state;
+
+    return (
+      <React.Fragment>
+        <h3>Description</h3>
+        <p>{operatorFieldDescriptions.spec.description}</p>
+        <div className="oh-operator-editor-form__field-section">
+          <MarkdownEditor
+            title="Details about the managed application"
+            description="Please describe the name, features, and use case(s) for your managed application briefly, and include links to read more about it."
+            markdown={application}
+            validationError={applicationError}
+            minHeadlineLevel
+            onChange={markdown => this.onUpdate('application', markdown)}
+            onValidate={() => this.validate('application')}
+          />
+          <MarkdownEditor
+            title="Details about the Operator"
+            description="Please describe the features your Operator provides to manage the application and what is it's value add."
+            markdown={about}
+            validationError={aboutError}
+            minHeadlineLevel
+            onChange={markdown => this.onUpdate('about', markdown)}
+            onValidate={() => this.validate('about')}
+          />
+          <MarkdownEditor
+            title="Details the prerequisites for enabling the Operator"
+            description="Please describe any steps a user needs to take prior to enabling this Operator (e.g. any Secrets or ConfigMaps that need to be in place upfront)."
+            markdown={prerequisites}
+            validationError={prerequisitesError}
+            minHeadlineLevel
+            onChange={markdown => this.onUpdate('prerequisites', markdown)}
+            onValidate={() => this.validate('prerequisites')}
+          />
+        </div>
+      </React.Fragment>
+    );
+  }
+}
 
 DescriptionEditor.propTypes = {
-  operator: PropTypes.object.isRequired,
```

Comment:
operator is still required. if this is removed for lint reasons we can add the disable lint for this line.

Reply:
Yes, linter was objecting against it